MyOwnORM/MyOwnORM/CustomKeyAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    public sealed class CustomKeyAttribute : Attribute
+    {
+        public string Id { get; }
```

Comment:
Why do you need this property? As I see, you apply it to property with PK, so what is it?

Reply:
This property is created to define special name for PK in model.

Reply:
You mean name of constraint or name of attribute in database?

Reply:
Yes

Reply:
Name of attribute in database